ironfish/src/network/peerNetwork.ts

```diff
@@ -1238,13 +1188,11 @@ export class PeerNetwork {
     return new GetCompactBlockResponse(block.toCompactBlock(), message.rpcId)
   }
 
-  private async handleNewBlockMessage(peer: Peer, message: NewBlockMessage) {
+  private async handleNewBlockMessage(peer: Peer, block: Block) {
```

Comment:
Maybe `handleRequestedBlock` now, since it's only used in the case where a block is directly requested by the block fetcher

Reply:
yep 👍🏻